packages/fiori3/src/components/FilterBar/FilterBar.jss.ts

```diff
@@ -0,0 +1,90 @@
+import { fonts, sap_fiori_3 } from '@fiori-for-react/styles';
```

Comment:
please don't import `sap_fiori_3` directly. all parameters are available in the `parameters` object of the function in line 4